src/server/game/Movement/MovementGenerators/HomeMovementGenerator.h

```diff
@@ -21,18 +21,15 @@
 
 #include "MovementGenerator.h"
 
-class Creature;
```

Comment:
and why it's not needed ? could you please elaborate more in detail why it was removed and why it shouldn't be needed ?